skipper.yaml

```diff
@@ -36,8 +36,14 @@ volumes:
   - /dev/:/dev
   - /run/udev:/run/udev
 
+  # podman - sharing the podman.socket between the host and the skipper container
+  - $XDG_RUNTIME_DIR/podman/podman.sock:/run/podman/podman.sock
```

Comment:
in assisted-service we have ``/run/podman/podman.sock:/run/podman/podman.sock``. what is it different?

Reply:
It depends on the podman-socket installation.